spec/models/ext_management_system_spec.rb

```diff
@@ -312,6 +312,40 @@
     end
   end
 
+  context "refresh" do
+    before do
+      @ems = FactoryBot.create(:ext_management_system)
+    end
```

Comment:
Can we use `let` for `ems` instead of the before block (we tend to be moving towards `let` for simple stuff)

Reply:
Done!